README.md

```diff
@@ -1,8 +1,31 @@
-## Generate something from GIR
+# GIR
+
+The `GIR` is used to generate both sys level and Rust-user API level.
+
+## Generate FFI (sys level)
+
+You first need to get the Gir file corresponding to the project you want to bind. You can get them from [here](https://github.com/gtk-rs/gir-files) or directly on [ubuntu website](http://packages.ubuntu.com/) (for example: http://packages.ubuntu.com/zesty/amd64/libgtk-3-dev).
+
+When you're ready, let's generate the FFI part. Don't forget to download the gir files as well.
+
+```shell
+cargo run --release -c YourGirFile.toml -d ../gir-files -m sys -o the-output-directory-sys
+```
+
+The generated files will be placed in `the-output-directory-sys`. You now have the sys part of your binding!
+
+A TOML file should have been generated as well (you can take a look to [gir-gtk.toml](https://github.com/gtk-rs/sys/blob/master/conf/gir-gtk.toml) as an example).
```

Comment:
Hm, 'TOML file` ??? May be you mean `Cargo.toml`? Then you partially right, it generated, if I not wrong